python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -124,13 +129,131 @@ def setDefaults(self):
+## \copybrief AssembleCoaddTask
+## \}
+
 class AssembleCoaddTask(CoaddBaseTask):
-    """Assemble a coadd from a set of coaddTempExp
+    """!
+\anchor AssembleCoaddTask
+
+\brief Assemble a coadded image from a set of coadded temporary exposures.
+
+\section pipe_tasks_assembleCoadd_Contents Contents
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Purpose
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Initialize
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Run
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Config
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Debug
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Example
+
+\section pipe_tasks_assembleCoadd_AssembleCoaddTask_Purpose	Description
+
+\copybrief AssembleCoaddTask_
+
+We want to assemble a coadded image from a set of coadded temporary exposures (coaddTempExps). We implicitly 
+assume that the coaddTempExps were all observed in the same filter band. Each input coaddTempExp 
+covers a patch on the sky and corresponds to a single run/visit/exposure of the covered patch. We provide the 
+task with a list of coaddTempExps (selectDataList) from which it selects coaddTempExps that cover the 
+specified patch (pointed at by dataRef). Each coaddTempExp that goes into a coadd will typically have an 
+independent photometric zero-point Therefore, we must scale each coaddTempExp to set it to a common 
+photometric zeropoint. By default, each coaddTempExp has backgrounds and hence will require 
+config.doMatchBackgrounds=True. When background matching is enabled, the task may be configured to 
+automatically select a reference exposure (config.autoReference=True). If this is not done, we require that 
+the input dataRef provides access to a coaddTempExp (dataset type coaddName + 'Coadd_tempExp') which is used 
+as the reference exposure. The coadd is computed as a mean with optional outlier rejection. Criteria for 
+outlier rejection are set in \ref AssembleCoaddConfig. Finally, coaddTempExps can have bad 'NaN' pixels which 
+received no input from the source calExps. We interpolate over these bad (NaN) pixels.
+
+  \par Inputs:
+    coaddTempExp{tract,patch,filter}: ExposureF
+  \par Outputs:
+    deepCoadd{tract,patch,filter}: ExposureF
+  \par Data Unit:
+    tract, patch, filter
+
+AssembleCoaddTask uses several sub-tasks. These are
+<DL>
+  <DT>\ref ScaleZeroPointTask_ "ScaleZeroPointTask"</DT>
+  <DD> create and use an image scaler object to scale the photometric zeropoint for each temp exposure</DD>
+  <DT>\ref MatchBackgroundsTask_ "MatchBackgroundsTask"</DT>
+  <DD> match background in a temp exp to a reference exposure (and select the reference exposure if one is 
+  not provided).</DD>
+  <DT>\ref InterpImageTask_ "InterpImageTask"</DT>
+  <DD>interpolate across bad pixels (NaN) in the final coadd</DD>
+</DL>
+You can replace these subtasks if you wish.
+
+\section pipe_tasks_assembleCoadd_AssembleCoaddTask_Initialize       Task initialization
```

Comment:
Does this need to be part of the overall summary?  I don't feel like it adds much.